proto/source_metadata.proto

```diff
@@ -188,6 +188,17 @@ message Teams {
   string email = 7;
 }
 
+// https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-FileInfo
+message Artifactory {
+  string repo = 1;
+  string path = 2;
+  string link = 3;
+  string created_at = 4;
+  string created_by = 5;
+  string modified_at = 6;
+  string modified_by = 7;
```

Comment:
are we able to capture an email (preferred) or a username of the person who uploaded the package?

Reply:
It looks like there's a [GetUserDetails](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-GetUserDetails) API that contains an email.

I'm not certain, but I think `created_by` and `modified_by` is the username (see [FileInfo docs](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-FileInfo)), so we can make another request to get the email.

Reply:
Maybe we can simplify the metadata to choose whichever timestamp is the latest, and have a `username` and `timestamp` instead of capturing both creating and modification data.

`email` can be added too as best effort

Reply:
We might be able to build a lookup from one request to Get Users List. I'm not sure if it contains email though.

https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-GetUsers

I think simplifying the metadata to `timestamp` and `username` is a good call.